src/Bot/commands/commands.ts

```diff
@@ -1,14 +1,25 @@
-import { SlashCommandBuilder } from "@discordjs/builders";
+import { SlashCommandBuilder } from '@discordjs/builders'
 
 const userSlashCommand =
-    new SlashCommandBuilder()
-        .setName("lookup")
-        .setDescription('Get the user Discord username by their C0D3 username')
-        .addStringOption(option =>
-            option.setName('username')
-                .setDescription("The user's username on C0D3")
-                .setRequired(true))
+  new SlashCommandBuilder()
+    .setName("lookup")
```

Comment:
Did the double quotes get past the ESlint/prettier rules? Just wondering becuase if put double quotes anywhere it lints red until I change it or format the file.

Reply:
Oh, nevermind. That rule is on the c0d3-app repo, not this one.

Reply:
Yeah. We've eslint there but not here.